main.py

```diff
@@ -601,6 +601,9 @@ def get_redirect_route(
     get_redirect_route(
         r'/memorycachehandler', release_coordinator.MemoryCacheHandler),
 
```

Comment:
Maybe drop this newline to clump the release_coordinator stuff together.

Reply:
Done